### PR TITLE
chore(package.json): using local docsify instead of npx

### PR DESCRIPTION
**Pull request by willmendesneto**
Removing npx call for docsify and using local package instead.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "build:esm": "babel src --out-dir build/esm --extensions \".ts,.tsx\" --source-maps inline --ignore src/__tests__",
     "build:types-cjs": "tsc --emitDeclarationOnly --project tsconfig.build.json --outDir build/cjs && cp src/index.js.flow build/cjs",
     "build:types-esm": "tsc --emitDeclarationOnly --project tsconfig.build.json --outDir build/esm && cp src/index.js.flow build/esm",
-    "docs": "npx docsify-cli serve docs",
+    "docs": "docsify serve docs",
```
Comment (rohan-deshpande) [inline package.json:47]: Ah okay so the command should just be `docsify` and not `docsify-cli` if you've got it locally installed? I actually wanted to remove it from the dev dependencies but if this works then that's fine too.

Comment (willmendesneto) [reply package.json:47]: Yeah, actually it's added as a devDependency already. So I'll leave for you to decide, in that case

**Comment by rohan-deshpande** [comment]
@willmendesneto so I remember why we chose to use npx now. Docsify has a security vulnerability in it that hasn't been fixed. Better to remove it from dev dependencies and leave the npx call for now.

File: package.json (hunk 1 above)
Comment (rohan-deshpande) [reply package.json:50]: https://github.com/atlassian-labs/react-resource-router/pull/13#issuecomment-638653545

**Comment by willmendesneto** [comment]
All good! Closing this pull request since it's not helpful. Thanks!

**Pull request closed without merging**